roles/ceph-osd/defaults/main.yml

```diff
@@ -123,9 +123,11 @@ osd_containerized_deployment: false
 osd_containerized_deployment_with_kv: false
 kv_type: etcd
 kv_endpoint: 127.0.0.1
+kv_port: 4001
 ceph_osd_docker_prepare_env: "OSD_FORCE_ZAP=1"
 ceph_osd_docker_username: ceph
 ceph_osd_docker_imagename: daemon
+ceph_osd_docker_tag: latest
```

Comment:
I think one should try to make things as clear as possible and separating each part of the docker images is such thing.